### ISSUE TITLE
HTTP error 421 occurred

### ISSUE THREAD

**Issue by Potturuhari** [abapGit/abapGit#6929]
HTTP error 421 occurred: SSL handshake with github.com:443 failed: SSSLERR_PEER_CERT_UNTRUSTED (-102) Peer's X.509 certificate (chain) validation failed (missing trust?) SapSSLSessionStartNB()==SSSLE
[SSL Certificate Error.docx](https://github.com/abapGit/abapGit/files/15303690/SSL.Certificate.Error.docx)

**Comment by mbtools** [comment]
You need to update certificates in STRUST

https://docs.abapgit.org/user-guide/setup/ssl-setup.html

**Comment by Potturuhari** [comment]
> You need to update certificates in STRUST
> 
> https://docs.abapgit.org/user-guide/setup/ssl-setup.html

Thaka for the response ..but we have installed all the information shared in the link.

I have identified missing the one of certificate in hierarchy in my Google chrome ..

Attaching googles
![Uploading Google Chrome .png…]()
 chrome certificate valid hierarchy screen shot for your reference .

**Comment by mbtools** [comment]
What's the output of the test program?
https://docs.abapgit.org/user-guide/setup/ssl-test.html

If it does not say "ok", follow the troubleshooting steps:
https://docs.abapgit.org/user-guide/setup/ssl-setup.html#troubleshooting

**Comment by Potturuhari** [comment]
Hi Team,

We were able to observe the same error even after enabling the document. Could you kindly let us know whether there is a method to connect through Teams and investigate the problem?

**Comment by francisco-milan** [comment]
@Potturuhari, I was not able to see the screenshot using link you provided ("Uploading Google Chrome"), try uploading it again, or provide your email and I'll try to help you.

**Comment by Potturuhari** [comment]
Hello Milan,
Thank you for the response and support.
If possible, lets connect Via teams, my id is harikrishna.potturu@accenture.com

**Comment by Potturuhari** [comment]
Thank you Milan your support its working fine now ..

**Comment by mbtools** [comment]
Maybe leave a short comment on how you fixed it so others can learn from it

**Comment by francisco-milan** [comment]
Hi Marc,

Following your recommendation, we checked the certificates in STRUST and noticed that at the time, 2 out of the 3 certificates were uploaded to the SAP system, and there was 1 certificate missing. After uploading the missing certificate, the issue was resolved.

Best regards,

Francisco Milan.

![image](https://github.com/abapGit/abapGit/assets/36721657/682f8afd-6361-4ba6-9fb8-cc7622c1139f)
